packages/common/src/lib/action/actionbar/actionbar.component.ts

```diff
@@ -143,11 +162,43 @@ export class ActionbarComponent implements OnDestroy, OnChanges {
     return this.horizontal;
   }
 
+  get heightCondition(): boolean {
+    if (this.withTitle === false) {
+      if ((this.store.count * 46) + 184 > document.documentElement.clientHeight) {
```

Comment:
What does +184 represents?

Reply:
for the moment, a sum of multiples fixed height button and margin. i will work on that